test/test_user_pref_sched.py

```diff
@@ -494,32 +495,65 @@ def test_task_4(self):
         task_repo.save(tasks[0])
         assert n_available_tasks_for_user(project, 500) == 2
 
-
     @with_context
-    def test_task_5(self):
```

Comment:
what happened to this test?

Reply:
seems, lost it with rebase; updated to bring it back. thanks.